virttest/utils_conn.py

```diff
@@ -743,7 +743,11 @@ def conn_recover(self):
                                             r"[\#\$]\s*$")
             libvirtd_service = utils_libvirtd.Libvirtd(session=session)
             if libvirt_version.version_compare(5, 6, 0, session):
+                process.run("systemctl stop libvirtd.socket")
+                libvirtd_service.stop()
                 session.cmd("systemctl stop libvirtd-tcp.socket")
+                process.run("systemctl daemon-reload")
```

Comment:
@cliping  it looks like you should execute new added comments in session. So 'process.run' should be 'session.cmd', right?

Reply:
Updated. Thanks.

Reply:
I remember that your original fix was stopping libvirtd.socket and reloading deamon in both conn_recover  and conn_setup. Is the updates in conn_recover() removed by mistake?